sys/src/9/port/lib.h

```diff
@@ -13,7 +13,7 @@
  */
 #define nelem(x)	(sizeof(x)/sizeof((x)[0]))
 #define offsetof(s, m)	(uint64_t)(&(((s*)0)->m))
-#define assert(x)	if(x){}else _assert(#x)
+#define assert(x)	do{if(!(x))_assert(#x)}while(0)
```

Comment:
you missed a semicolon before closing key:
do{if(!(x)) _assert(#x);}while(0)
Also an space between if and assert, just for reading.